pipeline/05-finalize.R

```diff
@@ -155,14 +155,46 @@ tryCatch(
     # Save an empty report so that this pipeline step produces the required
     # output even in cases of failure
     message("Saving an empty report file in order to continue execution")
-    sink(here("reports", "performance.html"))
+    sink(here("reports", "performance", "performance.html"))
```

Comment:
**[Thought, non-blocking]** I wonder if it would be better to make use of the `paths$output$report_performance$local` attribute here instead of hardcoding the path, so that we only have to change the output location in one place? Or perhaps there's something that prevents this approach?

```suggestion
    sink(paths$output$report_performance$local)
```